src/dra/73AAC.c

```diff
@@ -0,0 +1,83 @@
+#include "common.h"
+#include "dra.h"
+#include "objects.h"
+#include "sfx.h"
+
+void func_80113AAC(void) {
+    s32 var_s1 = 0;
+    s32 temp;
+
+    D_80072F6A[0]++;
+    if (func_8010FDF8(2) == 0) {
```

Comment:
I suggest you to do an early return `if (func_8010FDF8(2) != 0) { return; }` to avoid too much code indentation.